updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
 
 .Prerequisites
 
+* Have access to a conatiner registry in the disconnected environment to push and pull images. The container registry should be compatible with Docker registry API v2.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
```

Comment:
This is pretty grim, and we don't call it out in our [usual CLI update docs][1].  But if we want to punt and keep this existing wording for now, that's fine with me too.  Also not clear to me why this is over in `updating/` while the existing stuff is under `modules/`, but I'm ok punting on that as well ;).

[1]: https://github.com/openshift/openshift-docs/blame/4abd206b42d38e5bd66423d780c4c900b395e8ff/modules/update-upgrading-cli.adoc

Reply:
I think the generic idea behind modules is that if you expect to reuse some doc in multiple places it should be a module. May be we want to create a module for some of the docs here. Lets move forward and take a look a back after some iteration.